README.md

```diff
@@ -1,3 +1,28 @@
-# nullstone
+# Nullstone
 
-Nullstone is a developer platform to manage and launch apps on your own cloud.
+Nullstone is a Heroku-like developer platform launched on your cloud accounts.
+We offer a simple developer experience for teams that want to use Infrastructure-as-code tools like Terraform.
+
+This repository contains code for the Nullstone CLI which is used to manage Nullstone from the command line.
+This includes creating and deploying app, domains, and datastore as well as creating and managing Terraform workspaces.
+
+## Documentation
+
+For full documentation, visit [docs.nullstone.io](https://docs.nullstone.io).
+
+Visit [nullstone/README.md](nullstone/README.md) for CLI instructions.
+
+## Community & Support
+
```

Comment:
Ah, I see why you were trying to setup the Public Roadmap, Discussions, Issues, and Slack. This is super cool.